pages/common/wuzz.md

```diff
@@ -0,0 +1,19 @@
+# wuzz
+
+> Interactive HTTP Inspection.
```

Comment:
Needs slightly better wording. How about - "Tool to interactively inspect HTTP requests/responses." ?